plugins/kinetic-dp/fu-kinetic-dp-aux-dpcd.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2021 Jeffrey Lin <jlin@kinet-ic.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "fu-kinetic-dp-aux-dpcd.h"
+
+gboolean
+fu_kinetic_dp_aux_dpcd_read_oui(FuKineticDpConnection *connection, guint8 *buf, guint32 buf_size, GError **error)
+{
+    if (buf_size < DPCD_SIZE_IEEE_OUI)
+        return FALSE;
+
+    if (!fu_kinetic_dp_connection_read(connection, DPCD_ADDR_IEEE_OUI, buf, DPCD_SIZE_IEEE_OUI, error))
+    {
+        g_prefix_error (error, "Failed to read source OUI!");
```

Comment:
rather than end with !, can you please change to `: `.  This will read better when it prefixes